pkg/controller/installation/core_controller.go

```diff
@@ -265,20 +262,8 @@ func add(c controller.Controller, r *ReconcileInstallation) error {
 				// Create occurs because we've created it, so we can safely ignore it.
 				return false
 			},
-			UpdateFunc: func(e event.UpdateEvent) bool {
-				if utils.IgnoreObject(e.ObjectOld) && !utils.IgnoreObject(e.ObjectNew) {
-					// Don't skip the removal of the "ignore" annotation. We want to
-					// reconcile when that happens.
-					return true
-				}
-				// Otherwise, ignore updates to objects when metadata.Generation does not change.
-				return e.ObjectOld.GetGeneration() != e.ObjectNew.GetGeneration()
-			},
 		}
-		err = c.Watch(&source.Kind{Type: t}, &handler.EnqueueRequestForOwner{
```

Comment:
Doesn't this EnqueueRequestForOwner cause us to only run a reconcile when the secondary resource is owned by the Installation resource? Doesn't removing this mean we will receive updates and be triggered for all the secondaryResources which are cluster wide? That sounds very excessive.

Reply:
Maybe? I think a better way would be to update the watches to restrict to a namespace and/or object name, though. I'll look into that.

Reply:
By updating to use `AddNamespacedWatch` we now get predicates that match on specific name / namespace, so I think that resolves this concern.